lib/models/builder.js

```diff
@@ -26,7 +27,11 @@ class Builder extends CoreObject {
   constructor(options) {
     super(options);
 
+    this.broccoli2 = Boolean(options.broccoli2 === undefined ? experiments.BROCCOLI_2 : options.broccoli2);
```

Comment:
I think it would be better to remove `this.broccoli2` completely, and replace it with direct `experiments.BROCCOLI_2` checks...

Reply:
I couldn’t see how to mock that property in tests. I want to compare with the property on and off in the same test.

Reply:
Why? I think having tests around the expected behaviors should be enough. In general, from the outside, I don't think which builder we use should actually matter much, right?

Reply:
Well, we want to ensure that the files are exactly the same between both versions.
I guess if we have a fixture directory and compare against that, that'd also work.

Reply:
Totally agreed. I think the goal here should be that the migration is seemless and the output is identical. And we should be able to do that via fixtures (either inside the file system, or just via fixturify in the test itself)..

Reply:
Cool. Are we doing this anywhere already?